### PR TITLE
Update File Locations and SpeakerController.cs file Inclusion

### PR DESCRIPTION
**Pull request by pkellner**
Move files around per the "Contributing to Docs" doc.  Also update includes of speaker controller.  Did docfx build successfully but running on localhost 8080 did not show me what I expected. Just random titles and sections.

### CODE REVIEW

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/mvc/views/tag-helpers/intro.md](https://github.com/pkellner/Docs/blob/master/aspnetcore/mvc/views/tag-helpers/intro.md) | :white_check_mark:Succeeded | | [Details](#user-content-aspnetcore/mvc/views/tag-helpers/intro.md)


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C22%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201611220058287848-2229%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: aspnetcore/mvc/views/tag-helpers/intro.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ If you create a new ASP.NET Core web app named *AuthoringTagHelpers* (with no au
...
 
-To expose all of the Tag Helpers in this project (which creates an assembly named *AuthoringTagHelpers*), you would use the following:
+It is important to understand that if you author your Tag Helpers in the same project as your MVC web application (this project), that you must explicitly include the assembly name of this project (which is *AuthoringTagHelpers*).  If you do not include your current executing assembly in your *Views/_ViewImports.cshtml* file, your authored Tag Helpers will not be available in your view cshtml files.  Specifically, you would use the following:
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/views/tag-helpers/intro.md]: We decided this wasn't necessary. We don't like to use 
"It is important to understand"

**Comment by pkellner** [comment]
Hi @Rick-Anderson , Not sure of process to follow.  

- Feedback on my proposed file system layout here? 
- Suggestions/Comments on content, file inclusions, etc?  
- Will you be updating/accepting pull requests into my branch? (not sure best workflow for this, need advice)
- When I run docfx, I see my source includes but not same as online. How can I verify what it will actually look like?

**Comment by Rick-Anderson** [comment]
@pkellner this PR doesn't have any tag helper changes except for the "it's important" comment we previously declined. 

When we review your PR we'll provide feedback/suggestions/comments. DocFX should look pretty close to the live version except for styling (tables, etc).

File: aspnetcore/mvc/views/tag-helpers/intro.md (hunk 1 above)
Comment (pkellner) [inline aspnetcore/mvc/views/tag-helpers/intro.md]: @Rick-Anderson somehow, I'm getting confused between pull requests.  The diff's above show my first PR that had to do with _ViewImports.cshtml.  I'm good with closing and abandoning that one.  But, the title on top of my web page says "Update File Locations and SpeakerController.cs" which I issues yesterday and that had small changes to the previous pull request that was all about tag helpers.

Comment (Rick-Anderson) [inline aspnetcore/mvc/views/tag-helpers/intro.md]: @pkellner  tap on Files changed to see the net effect of all your commits.  It's probably easier to close this PR and create a new PR with the changes you want to submit.

capture

**Comment by pkellner** [comment]
OK, I'll create another pull request after closing this one.

**Pull request closed without merging**